src/app/model/enums/timestamp-response-status.ts

```diff
@@ -0,0 +1,5 @@
+export enum TimestampResponseStatus {
+    ACCEPTED = "ACCEPTED",
+    REJECTED  = "REJECTED",
+    CREATE = "CREATE"
```

Comment:
Use the same tempus for these (`ACCEPT` + `REJECT` or `CREATED`).

Given its use, I would go with `ACCEPT`, `REJECT` and `CREATE`.

Reply:
I agree, will update this.